tests/test_cliCmdTestIngest.py

```diff
@@ -35,36 +35,38 @@
 
 
 class IngestRawsTestCase(CliCmdTestBase, unittest.TestCase):
+    """Test the ingest-raws command-line tool."""
+
     mockFuncName = "lsst.obs.base.cli.cmd.commands.script.ingestRaws"
 
     @staticmethod
     def defaultExpected():
-        return dict(
-            config={},
-            config_file=None,
-            ingest_task="lsst.obs.base.RawIngestTask",
-            locations=(),
-            output_run=None,
-            processes=1,
-            regex=fits_re,
-            transfer="auto",
-            track_file_attrs=True,
-            fail_fast=False,
-        )
+        return {
+            "config": {},
+            "config_file": None,
+            "ingest_task": "lsst.obs.base.RawIngestTask",
+            "locations": (),
+            "output_run": None,
+            "processes": 1,
+            "regex": fits_re,
+            "transfer": "auto",
+            "track_file_attrs": True,
+            "fail_fast": False,
+        }
```

Comment:
It comes from the C4 checker at https://beta.ruff.rs/docs/rules/#flake8-comprehensions-c4

I agree that sometimes the dict() form is nicer because you don't have to quote the keywords.

Reply:
Can we disable that particular check?

Reply:
It's not on by default. Like "B" and "UP", I turned it on temporarily to see what it complained about.